src/extension/features/budget/display-total-monthly-goals/index.js

```diff
@@ -0,0 +1,109 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { getEmberView } from 'toolkit/extension/utils/ember';
+import { formatCurrency } from 'toolkit/extension/utils/currency';
+import { getCurrentRouteName } from 'toolkit/extension/utils/ynab';
+
+export class DisplayTotalMonthlyGoals extends Feature {
+  shouldInvoke() {
+    return getCurrentRouteName().indexOf('budget') !== -1 && this.settings.enabled !== '0';
+  }
+
+  extractCategoryGoalInformation(element) {
+    const emberId = element.id;
+    const viewData = getEmberView(emberId).data;
+    const {
+      subCategory,
+      monthlySubCategoryBudgetCalculation
+    } = viewData;
+
+    const goalType = subCategory.get('goalType');
+    const monthlyFunding = subCategory.get('monthlyFunding');
+    const targetBalanceDate = monthlySubCategoryBudgetCalculation.get('goalTarget');
+
+    let monthlyGoalAmount = 0;
+
+    switch (goalType) {
+      case 'MF': {
+        monthlyGoalAmount = monthlyFunding;
+        break;
+      }
+      case 'TBD': {
+        monthlyGoalAmount = targetBalanceDate;
+        break;
+      }
+    }
+
+    return {
+      monthlyGoalAmount,
+      isChecked: $(element).is('.is-checked')
```

Comment:
I think this is actually available on the `viewData` object. `viewData.get('isChecked');`